src/models/disassemblyoutput.cpp

```diff
@@ -265,7 +265,15 @@ DisassemblyOutput DisassemblyOutput::disassemble(const QString& objdump, const Q
     DisassemblyOutput disassemblyOutput;
     disassemblyOutput.symbol = symbol;
     if (symbol.symbol.isEmpty()) {
-        disassemblyOutput.errorMessage = QApplication::tr("Empty symbol ?? is selected");
+        disassemblyOutput.errorMessage =
```

Comment:
this doesn't really help - if you really want to wrap the `??` then just do that directly, no need for `.arg()`

Reply:
That's what I did, just to get a compile warning about trigraphs., which was the reason to split that. Another option is to use inline hex values, bit that is likely not useful for the translation.